### PR TITLE
Add issue template for local meetup organisers

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
This PR updates the issue template for tracking and managing local meetup organization tasks. The new template aims to provide a clear and structured way for organizers to detail their event plans, ensuring all necessary information is included and that the process is streamlined.

### CODE REVIEW

**Review by sunya-ch — COMMENTED** [review]
Thank you for pushing this. I believe it will be helpful for local organizer to onboard the sustainability week event. Leave some comments.

File: .github/ISSUE_TEMPLATE/10-action copy.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
+name: Local Meetup Tracking/Action Item
+description: Use this issue type to track and manage local meetup organization tasks. Avoid using this issue type for tasks expected to be closed within hours.
+title: "[<Tracking>/<Action>] Cloud Native Sustainability Week 2024 <Meetup Location>"
```
[the remark this answers is not in the document]
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/10-action copy.yml:3]: 👍 we can add a `<YEAR>` field

[the remark this answers is not in the document]
Comment (leonardpahlke) [reply .github/ISSUE_TEMPLATE/10-action copy.yml:3]: 👍

File: .github/ISSUE_TEMPLATE/10-action copy.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    type: textarea
+    attributes:
+      label: To-Do
+      description: What specific actions are needed to organize this meetup?
+      value: |
+        1. [Local organizer] Review the [Meetup Organizer Guide](https://docs.google.com/document/d/1xfUqxt0EeMhrsg8h5p_cAJ1ZHzWryrIywue53rVRFJw/edit?usp=sharing) for onboarding.
+        2. [Local organizer] Check the [website](https://tag-env-sustainability.cncf.io/events/cloud-native-sustainability-week/).
+        3. [Local organizer] Create a PR to update the Meetup's table on the [website](https://tag-env-sustainability.cncf.io/events/cloud-native-sustainability-week/). Example [PR](https://github.com/cncf/tag-env-sustainability/pull/170).
+        4. [TAG ENV Lead] Assign lead organizer/co-lead to the issue.
+        5. Tag leads & co-leads if assistance is needed.
```
Comment (leonardpahlke) [inline .github/ISSUE_TEMPLATE/10-action copy.yml:44]: we dont need these two steps
```suggestion
```

File: .github/ISSUE_TEMPLATE/10-action copy.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    type: textarea
+    attributes:
+      label: Details
+      description: What are the goals and activities planned for the meetup?
+    validations:
+      required: true
+  - id: todo
+    type: textarea
+    attributes:
+      label: To-Do
```
Comment (leonardpahlke) [inline .github/ISSUE_TEMPLATE/10-action copy.yml:37]: these steps look like checkboxes. (see example CoC)

```
    type: checkboxes
    attributes:
      label: Code of Conduct
      description: By submitting this issue, you agree to follow our [Code of Conduct](https://www.cncf.io/conduct/).
      options:
        - label: I agree to follow this project's Code of Conduct
          required: true
```

File: .github/ISSUE_TEMPLATE/10-action copy.yml [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      description: What specific actions are needed to organize this meetup?
```
Comment (leonardpahlke) [inline .github/ISSUE_TEMPLATE/10-action copy.yml:38]: We could add potentially

- Joined one of the TAG ENV Sustainability Week Sync to talk about your event
- Found a sponsor for the event
- Found a location for the event
- Set a date for the event
- Announced the event
- Reached out to the Cloud Native Sustainability Week Organizer team to promote the event

**Comment by leonardpahlke** [comment]
Added a couple of comments @Nancy-Chauhan - thanks! this looks great

**Review by leonardpahlke — APPROVED** [review]
LGTM, lets refine it over time!

**Pull request merged**